### PR TITLE
export file

### PR DESCRIPTION
**Pull request by Haaankkk**
NOT ready to merge

JSON file stuff (input/output)

### CODE REVIEW

File: NeuroRuler/GUI/main.py [hunk 1]
```diff
@@ -847,6 +851,49 @@ def export_curr_slice_as_img(self, extension: str) -> None:
...
+        "z_rotation": global_vars.THETA_Z,
+        "slice": global_vars.SLICE,
+        "smoothing_conductance": global_vars.CONDUCTANCE_PARAMETER,
+        "smoothing_iterations": global_vars.SMOOTHING_ITERATIONS,
+        "smoothing_time_step": global_vars.TIME_STEP,
+        "circumference": circumference
+        }
+
+        with open(path, 'w') as outfile:
+            json.dump(data, outfile)
```
Comment (eric-unc) [inline NeuroRuler/GUI/main.py:886]: This seems like something that would be helpful in the CLI too btw

File: NeuroRuler/GUI/main.py [hunk 2]
```diff
@@ -847,6 +851,49 @@ def export_curr_slice_as_img(self, extension: str) -> None:
...
+
+    def get_circumference(self) -> str:
+        """Get the value on the "circumference_label"
+
+        :return: `str`"""
+        circumference_label: str = self.circumference_label.text()
+        split_parts = re.split('[:(]', circumference_label)
+        circumference = split_parts[1].strip()
+        return circumference
```
Comment (eric-unc) [inline NeuroRuler/GUI/main.py:895]: Is there a less hacky way to store the circumference data?

Comment (Haaankkk) [reply NeuroRuler/GUI/main.py:895]: Good point. Let me improve this in the next commit.

**Comment by Haaankkk** [comment]
Finished import and export
Did not know how to set the limit of the number of imported JSON files to 1.

**Comment by Haaankkk** [comment]
Fixing some minor bugs rn

**Comment by jesse-wei** [comment]
I improved export JSON. Import JSON is bugged. Doesn't seem to change smoothing options or threshold options. However, project is due in 4 minutes so yeah I'm merging and uploading this to pip.

**Comment by Haaankkk** [comment]
> I improved export JSON. Import JSON is bugged. Doesn't seem to change smoothing options or threshold options. However, project is due in 4 minutes so yeah I'm merging and uploading this to pip.

Yeah I found that bug as well. I know how to fix that bug. Hopefully this will not influence our whole project? I will fix it later.

**Comment by jesse-wei** [comment]
> > I improved export JSON. Import JSON is bugged. Doesn't seem to change smoothing options or threshold options. However, project is due in 4 minutes so yeah I'm merging and uploading this to pip.
> 
> Yeah I found that bug as well. I know how to fix that bug. Hopefully this will not influence our whole project? I will fix it later.

I fixed it.

**Comment by Haaankkk** [comment]
> > > I improved export JSON. Import JSON is bugged. Doesn't seem to change smoothing options or threshold options. However, project is due in 4 minutes so yeah I'm merging and uploading this to pip.
> > 
> > 
> > Yeah I found that bug as well. I know how to fix that bug. Hopefully this will not influence our whole project? I will fix it later.
> 
> I fixed it (#82).

Got it. Thanks!

**Pull request merged**